CMakeLists.txt

```diff
@@ -292,3 +292,9 @@ add_custom_target(clean-built
    COMMAND ${CMAKE_COMMAND} -E remove_directory ${PROJECT_SOURCE_DIR}/built
    COMMENT "Cleaning `built` directory"
 )
+
+# Debian Package
+if(UNIX)
```

Comment:
Personally I like KISS.  Unless/until we have reasonable expectation of supporting another platform in the near-ish future, we can just assume it's Linux.